patients/add_edit_event_user.php

```diff
@@ -581,17 +581,17 @@
 
 $statuses = array(
 '-' => '',
-'*' => xl('* Reminder done'),
-'+' => xl('+ Chart pulled'),
-'x' => xl('x Cancelled'), // added Apr 2008 by JRM
-'?' => xl('? No show'),
-'@' => xl('@ Arrived'),
-'~' => xl('~ Arrived late'),
-'!' => xl('! Left w/o visit'),
-'#' => xl('# Ins/fin issue'),
-'<' => xl('< In exam room'),
-'>' => xl('> Checked out'),
-'$' => xl('$ Coding done'),
+'*' => xlt('* Reminder done'),
+'+' => xlt('+ Chart pulled'),
+'x' => xlt('x Cancelled'), // added Apr 2008 by JRM
+'?' => xlt('? No show'),
+'@' => xlt('@ Arrived'),
+'~' => xlt('~ Arrived late'),
+'!' => xlt('! Left w/o visit'),
+'#' => xlt('# Ins/fin issue'),
+'<' => xlt('< In exam room'),
+'>' => xlt('> Checked out'),
+'$' => xlt('$ Coding done'),
```

Comment:
It's generally best to do the html escaping where the data is outputted to html, so would not escape here and would instead escape the output of the variable(s). That being said, it looks like this array is not even used in this script, so can remove it.